oneflow/python/test/ops/test_module.py

```diff
@@ -57,6 +57,16 @@ def test_relu(test_case):
         y = relu(x)
         print(y.numpy())
 
+    def test_tanh(test_case):
+        input_arr = np.array([-1, 0, 1]).astype(np.float32)
+        x = flow.Tensor(input_arr)
+
+        tanh = flow.nn.Tanh()
+        y = tanh(x)
+        z = np.tanh(input_arr)
+        
+        test_case.assertTrue(np.allclose(y.numpy(), z, atol=1e-4))
```

Comment:
```suggestion
        test_case.assertTrue(np.allclose(y.numpy(), z, rtol=1e-4, atol=1e-4 ))
```
相对和绝对误差都设置，保险一点（到底设多少，根据测试情况自己定一下，suggestion里的应该是可以的。